### DISCUSSION TITLE
Learn c++ first before Carbon?

### DISCUSSION THREAD

**Discussion by Pgirardin** [carbon-language/carbon-lang#1634] [General]
Learn c++ first before Carbon?

**Comment by pacion** [comment]
I'd recommend that, learning any language makes learning another language much much easier because of the concepts that are repeatable.

Let's look on the other hand, if you really want to start Carbon - feel free to use it, or at least consider playing need for speed Carbon :D

**Reply by Preciouskings** [reply]
Yes, very compulsory

**Reply by anton-roos** [reply]
Whahaha Need for Speed Carbon 🤣

**Reply by cmedamine** [reply]
haha great game, i still playing it

**Comment by jonmeow** [comment]
Just to be clear, Carbon is not ready for use:
https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/faq.md#how-soon-can-we-use-carbon

**Comment by YagaoDirac** [comment]
Now the language is an experiment. It's not a tool yet. If you want a tool better than c++ and you are gonna use it, Carbon is not a choice now. The answer depends on your purpose and other tools you are gonna use in the same project.
